echo-model/src/main/java/com/netflix/spinnaker/echo/model/Trigger.java

```diff
@@ -80,7 +80,7 @@ public String toString() {
    */
   String subscriptionName;
   String pubsubSystem;
-  List<ExpectedArtifact> expectedArtifacts;
+  List<String> expectedArtifacts;
```

Comment:
Should we call these `expectedArtifactIds`?

Reply:
Yeah, let me rename.

Reply:
Changed this everywhere and re-tested, should be ok now.